src/NzbDrone.Core/Extras/ExtraService.cs

```diff
@@ -48,9 +52,32 @@ public class ExtraService : IExtraService,
             _diskProvider = diskProvider;
             _configService = configService;
             _extraFileManagers = extraFileManagers.OrderBy(e => e.Order).ToList();
+            _albumExtraManager = albumExtraManager;
             _logger = logger;
         }
 
+        /// <summary>
+        /// Import extra files belonging to an album. Track-specific extras will be ignored.
+        /// </summary>
+        /// <param name="importedTracks">The imported tracks in the source dir.</param>
+        /// <param name="destinationDir">Destination dir for the album extras</param>
```

Comment:
Blitz the method comments, we try and avoid it with the codebase (our style)

Reply:
Done.